examples/multi-insert-relative.rs

```diff
@@ -0,0 +1,75 @@
+use indicatif::{MultiProgress, ProgressBar, ProgressFinish, ProgressStyle};
```

Comment:
One of my pet peeves is that indicatif already has too many examples. Do you think you could modify one of the existing examples to use `insert_before()` and/or `insert_after()`?

Reply:
Sure, I can modify the "multi.rs" example and will do so now. As I alluded in the previous version of this pull request, the other multi-* examples are too complicated to change to use `insert_before`/`insert_after`.

Reply:
Done